spec/definitions/GroupReport.yaml

```diff
@@ -0,0 +1,26 @@
+type: object
```

Comment:
This definition is same as DeviceReport, right?

Reply:
Yes right.